apps/prairielearn/src/pages/instructorRequestCourse/instructorRequestCourse.html.ts

```diff
@@ -54,7 +72,22 @@ export function RequestCourse({
         })}
         <main id="content" class="container">
           <h1 class="sr-only">Request a Course</h1>
-          ${CourseRequestsCard({ rows })}
+          ${CourseRequestsCard({ rows })} ${EncodedData(lti13Info, 'courseRequestLti13Info')}
+          ${Modal({
+            id: 'lti13FillModal',
+            title: html`Auto-fill with ${lti13Info?.['cr-institution'] ?? 'LMS'} data?`.toString(),
```

Comment:
The `Modal` component already renders and escapes this correctly.

```suggestion
            title: `Auto-fill with ${lti13Info?.['cr-institution'] ?? 'LMS'} data?`,
```